rust-runtime/aws-smithy-http-server-python/examples/pokemon_service.py

```diff
@@ -78,6 +79,9 @@ def value(self) -> int:
 #   https://docs.python.org/3/library/multiprocessing.html#sharing-state-between-processes
 @dataclass
 class Context:
+    # Inject Lambda context if service is running on Lambda
+    lambda_ctx: LambdaContext = None
```

Comment:
```suggestion
    lambda_ctx: Optional[LambdaContext] = None
```

Reply:
Our check will fail if we wrap it with `Optional`: https://github.com/awslabs/smithy-rs/pull/1985/files#diff-808ab7f16108679c85b347d9b60605bb96a6fad0f1e6ce86ba679bc922fd105eR100 I'll update PR to support `Optional` types. It'd be better if we enforce users to wrap their types with `Optional` always as it indicates that there is no compile-time guarantee that this value if provided and they should always check at runtime to make sure that value is provided, what do you think?

Reply:
Yep, I agree.

Reply:
Addressed in https://github.com/awslabs/smithy-rs/pull/1985/commits/d82ea4e9b8d8169f81ca31bf1444eb32e808f091